lib/logger.js

```diff
@@ -2,66 +2,62 @@
 
 "use strict";
 
-class Logger {
-    constructor(config) {
-        this.config = config;
-    }
-
-    /**
-     * Displays a message to the console only.
-     *
-     * @param {*} ...data All parameters are sent to console.log.
-     */
-    console() {
-        var args;
-
-        args = [].slice.call(arguments);
-        console.log.apply(console, args);
-    }
-
-
-    /**
-     * Logs debug statements to stderr only if
-     * debug has been turned on in environment variables
-     * or config.
-     *
-     * @param {string} text
-     */
-    debug(text) {
-        if (this.config.debug) {
-            console.error("DEBUG: " + text);
+module.exports = (config) => {
+    return {
+        /**
+         * Displays a message to the console only.
+         *
+         * @param {*} ...data All parameters are sent to console.log.
+         */
+        console: function (x) {
```

Comment:
Do we need x? You don't do anything with it. Might have been left over when explaining the situation.